src/script/lua_api/l_object.cpp

```diff
@@ -1419,34 +1553,55 @@ int ObjectRef::l_set_physics_override(lua_State *L)
 	if (playersao == nullptr)
 		return 0;
 
-	if (lua_istable(L, 2)) {
-		bool modified = false;
-		modified |= getfloatfield(L, 2, "speed", playersao->m_physics_override_speed);
-		modified |= getfloatfield(L, 2, "jump", playersao->m_physics_override_jump);
-		modified |= getfloatfield(L, 2, "gravity", playersao->m_physics_override_gravity);
-		modified |= getboolfield(L, 2, "sneak", playersao->m_physics_override_sneak);
-		modified |= getboolfield(L, 2, "sneak_glitch", playersao->m_physics_override_sneak_glitch);
-		modified |= getboolfield(L, 2, "new_move", playersao->m_physics_override_new_move);
-		if (modified)
-			playersao->m_physics_override_sent = false;
-	} else {
+	playersao->m_physics_override_set = false;
+
+	if (read_physics_modifier(L, 2, playersao->m_physics_override)) {
+		if (has_field(L, 2, "sneak") ||
+				has_field(L, 2, "sneak_glitch") ||
+				has_field(L, 2, "new_move")) {
+			log_deprecated(L, "Setting physics flags in set_player_overrides is deprecated, use set_physics_flags instead");
+
+			playersao->m_physics_override_sneak = getboolfield_default(
+					L, 2, "sneak", playersao->m_physics_override_sneak);
+			playersao->m_physics_override_sneak_glitch = getboolfield_default(
+					L, 2, "sneak_glitch", playersao->m_physics_override_sneak_glitch);
+			playersao->m_physics_override_new_move = getboolfield_default(
+					L, 2, "new_move", playersao->m_physics_override_new_move);
+		}
+
+		playersao->m_physics_override_sent = false;
+		playersao->m_physics_override_set = true;
+	} else if (lua_gettop(L) > 2 || !lua_isnil(L, 2)) {
 		// old, non-table format
 		// TODO: Remove this code after version 5.4.0
```

Comment:
what does "after" mean here? :thinking:  :thinking:  :thinking:

Reply:
This is from Zughy, I imagine he meant after 5.4.0 is released